### PR TITLE
Include LICENSE in compiled jar

### PR DESCRIPTION
**Pull request by pahimar**
Also added a licenses block to the pom

### CODE REVIEW

File: src/main/resources/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Microsoft Corporation
```
Comment (ProbablePrime) [inline src/main/resources/LICENSE]: Drop the year as per MSFT guidelines

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Beam Interactive, Inc.
+Copyright (c) 2016 Microsoft Corporation
```
Comment (ProbablePrime) [inline LICENSE]: Drop the year as per MSFT guidelines

**Review by ProbablePrime — CHANGES_REQUESTED** [review]

**Review by connor4312 — APPROVED** [review]

**Comment by connor4312** [comment]
@ProbablePrime please follow up!

**Comment by ProbablePrime** [comment]
Ahh sorry @pahimar you can re-request a review once you've changed stuff and that'll notify me. :D

**Review by ProbablePrime — APPROVED** [review]

**Comment by connor4312** [comment]
By default, Github will notify you when new changes are pushed to a PR that you're watching

**Comment by ProbablePrime** [comment]
I did not notice >.< my bad!

**Pull request merged**